### ISSUE TITLE
Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>.

### ISSUE THREAD

**Issue by Arpels** [couds/react-bulma-components#44]
Hello, I don't know if this has been identified yet, but when you put a button inside an Navbar.Item, this error pops-up.

```
<Navbar.Item>
  <Button color="primary">About</Button>
</Navbar.Item>
```
Regards,

**Comment by couds** [comment]
Hi @Arpels  thanks for reporting this. 

If you want to add a button inside a Navbar.Item I recommend to change the the Navbar.Item to a div (`<Navbar.Item renderAs="div" ...`) bulma.io use div for this kind of items in their docs. https://bulma.io/documentation/components/navbar/#navbar-item
 
Another possible solution will be change the button from `a` element to a `button` (you can do it also using the `renderAs` prop of the button), personally I prefer to display the buttons as anchor elements but I'm thinking on change the default to a `button`  depending of what the community says.

Thanks, and let me know if this help.

**Comment by LaRuaNa** [comment]
Hi @Arpels, 
the problem is both [Navbar.Item](https://couds.github.io/react-bulma-components/?selectedKind=Navbar&selectedStory=Default&full=0&addons=0&stories=1&panelRight=0) and [Button](https://couds.github.io/react-bulma-components/?selectedKind=Button&selectedStory=By%20Colors&full=0&addons=0&stories=1&panelRight=0) are rendered as `a` tag. As a workaround you can use the renderAs prop and render Navbar.Item as `div`. But that needs to fixed.

**Comment by couds** [comment]
solved with d79a28b767364d58d2175aa575603aaf1f05a7e4 (Now buttons are rendered as buttons instead of anchors)
